README.md

```diff
@@ -457,6 +457,7 @@ Hurray! You successfully made a contribution! 🎉
 
   - [R Varshith Kumar](https://github.com/codefromrvk)
   - [Rahman Haroon](https://github.com/rahmanharoon)
+  - [Rahul Kumar Roy](https://github.com/mRahulRoy)
   - [Rahul Aggarwal](https://github.com/rahul-aggarwal000005)
```

Comment:
Add your name below `Rahul Aggarwal`

Reply:
Sir i am new in these. I hope you understand. Thanks for helping me in this first contribution.